aws_cloudwatch.go

```diff
@@ -483,6 +485,9 @@ func detectDimensionsByService(service *string, resourceArn *string, fullMetrics
 	switch *service {
 	case "alb":
 		dimensions = queryAvailableDimensions(arnParsed.Resource, getNamespace(service), fullMetricsList)
+		dimensions = queryAvailableDimensions(arnParsed.Resource, getNamespace(service), clientCloudwatch)
```

Comment:
This looks like an error? - Beside that looks good. Would like to add this :) - Could you remove this line if not needed or explain it. Thank you!

Reply:
@thomaspeitz This has been removed.